drivers/filesystems/btrfs/btrfs.c

```diff
@@ -627,6 +668,9 @@ static bool lie_about_fs_type() {
     INIT_UNICODE_STRING(fsutil, L"FSUTIL.EXE");
     INIT_UNICODE_STRING(storsvc, L"STORSVC.DLL");
 
+    /* Not doing a Volkswagen, honest! Some IFS tests won't run if not recognized FS. */
```

Comment:
lmao 😂 
Btw here I'm attaching URLs about that "IFSTest" thingie for reference purposes:
https://docs.microsoft.com/en-us/previous-versions/windows/hardware/wlk/ff563405(v=vs.85)
https://docs.microsoft.com/en-us/windows-hardware/test/hlk/testref/14b230f3-7eee-437e-ab2f-375b200de6f3
https://docs.microsoft.com/en-us/windows-hardware/test/hlk/testref/e84242de-56a9-4e6b-8f8c-ad3f7252cec7